charts/graph-network-indexer/Chart.yaml

```diff
@@ -15,11 +15,11 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.2.7
+version: 1.0.0
```

Comment:
we don't have (yet) a release versioning policy / standard. Making this one a 1.0 kinda stands out from other charts, and feels a bit specially weird coupled with potentially a transient/non-release tag below. Personally would vote for a 0.3.0 or 0.5.0 - but again, we have no consistent policy on this yet, so just voicing my subjective opinion :p your call